torch/nn/modules/module.py

```diff
@@ -574,6 +574,12 @@ def to(self, *args, **kwargs):
             Parameter containing:
             tensor([[ 0.1913, -0.3420],
                     [-0.5113, -0.2325]])
+            >>> linear.to(torch.cdouble)
```

Comment:
Let's make this a complete, separate complex example with a complex input and a complex output.

"Example::" -> "Examples::" to support the new example.